middleware/middleware-1.3.23.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: /release-notes/middleware/1.3.23
+title: Version 1.3.23
+---
+
+# fiskaltrust.Middleware 1.3.23 (Germany)
+_September 15, 2021_
+
+In this version of the Middleware, we added the possibility to easily identify mandatory and optional signature items when printing receipts. Additionally, the Middleware is now able to automatically update the _Fiskal Cloud Connector_, and several bugs and stability issues have been resolved.
+
+
+:::caution
```

Comment:
Is this being replaced somewhere?

Reply:
Yep, this is handled by Docusaurus (https://docusaurus.io/docs/markdown-features/admonitions)